tasks/pip.yml

```diff
@@ -120,3 +120,13 @@
     virtualenv_python: "{{ openwisp2_python }}"
     virtualenv_site_packages: yes
   notify: reload supervisor
+
+- name: Install CORS
+  when: "{{ cors:enabled }}"
```

Comment:
this way of accessing dictionary keys doesn't look correct. Have you tested the patch?

Reply:
@nemesisdesign I had run the playbook with CORS enabled and disabled both, and the server was hosted in both the cases. I don't know how to test CORS so that's still a doubt.

Reply:
it shouuld be `cors.enabled`, same applies for all the other lines

Reply:
Ok, I will test it tomorrow with the correction again.